src/containers/Login/index.tsx

```diff
@@ -33,11 +44,21 @@ const Login = () => {
         onSuccess: (data) => {
           if (data) {
             setCookie(cookieName, data.data.data.access_token);
-            navigate(`/`);
+            if (data.data.data.role === 'Employee') {
```

Comment:
```suggestion
            if (data.data.data.role === 'Employee') {
```

  if (data?.data?.data?.role === 'Employee') {